nj2k/src/org/jetbrains/kotlin/nj2k/JavaToJKTreeBuilder.kt

```diff
@@ -719,8 +719,9 @@ class JavaToJKTreeBuilder constructor(
             visibility(referenceSearcher) { ast, psi -> ast.withFormattingFrom(psi) }
 
         fun PsiField.toJK(): JKField {
+            val typeElement = typeElement
```

Comment:
Why we need this variable?

Reply:
`this.typeElement` calls: 
https://github.com/JetBrains/intellij-community/blob/master/java/java-psi-impl/src/com/intellij/psi/impl/source/PsiFieldImpl.java#L132-L139

We should avoid unnecessary method calls.
https://github.com/t-kameyama/kotlin/blob/07054bdee360b173175d4df6ade67a209ad504cd/nj2k/src/org/jetbrains/kotlin/nj2k/JavaToJKTreeBuilder.kt#L724

Reply:
This call is very cheap, so there is no need for that :)

But for avoiding this you can move all functionality related to working with type elements to separated function like `PsiTypeElement.toJk(): JKtypeElement` which will retrieve type, annotations, and formating from type element